dashboards/apache-airflow/README.md

```diff
@@ -0,0 +1,10 @@
+### Dashboards for Apache Airflow Scheduler
+
+#### Notes
+
+- This dashboard is based on metrics [collected using the statsd_exporter](https://www.redhat.com/en/blog/monitoring-apache-airflow-using-prometheus).
+
+|Apache Airflow Scheduler|
```

Comment:
Please add a metadata.yaml file with an entry for this dashboard. You can find documentation for this file here https://github.com/GoogleCloudPlatform/monitoring-dashboard-samples#metadatayaml-content